src/GraphQL/Types/Composite/AutoRegisteringObjectGraphType.cs

```diff
@@ -113,7 +113,7 @@ private static string GetPropertyName<TSourceType>(Expression<Func<TSourceType,
             if (expression.Body is MemberExpression m1)
                 return m1.Member.Name;
 
-            if (expression.Body is UnaryExpression u && u.Operand is MemberExpression m2)
+            if (expression.Body is UnaryExpression {Operand: MemberExpression m2} u)
```

Comment:
I know this syntax is becoming fashionable for some developers. However, I would not want to use it, at least not yet. It is not easier to read than the current version. @Shane32  ?

Reply:
Agreed